app/services/get-breadcrumbs.js

```diff
@@ -20,6 +20,11 @@ module.exports = function (id, organisationLevel) {
     breadcrumbs.push(new Link(tree[reference].name, linkGenerator.fromReference(reference)))
     reference = tree[reference].parent
   } while (reference !== undefined)
+
+  if (breadcrumbs.length < 2) {
+    breadcrumbs = []
+  }
```

Comment:
Yea, there was no need to have a label saying `NPS` when there was no possible interaction with the breadcrumbs.